src/Misc/Master.cpp

```diff
@@ -512,6 +512,14 @@ static const Ports master_ports = {
         } else
             d.reply(d.loc, "i", m->time.tempo);
         rEnd},
+    {"sync::i:c:S", rProp(parameter) rDefault(0) rShort("Sync") rDoc("Syncronisation Source") rOptions(SYNCSOURCES), 0,
```

Comment:
The port tells us that this will accepts symbols ('S'), but it only accepts integers.  An `if` for symbols should be added (or if possible, the `rOption` macro should be used - it does that automatically).